server/controllers/ArticleController.js

```diff
@@ -75,6 +76,34 @@ class ArticleController {
         }
       }));
   }
+
+  /**
+   * controller to create an article
+   * @param {object} req
+   * @param {object} res
+   * @returns {void}
+   */
+  static fetchAllArticles(req, res) {
+    Article
+      .findAndCountAll({
+        include:
+          [{
+            model: User,
+            attributes: ['fullName', 'avatarUrl']
+          }]
+      })
+      .then((result) => {
+        res.status(200).json({
+          Articles: result.rows
+        });
+      })
+      .catch(err => res.status(500)
+        .json({
+          errors: {
+            message: [err.message]
```

Comment:
I think err.message might not be descriptive enough. I suggest you replace that with a descriptive statement such `Something went wrong. Try again or contact the administrator`. Aside from that, every other thing looks good to me.

Reply:
Okay